packages/alfa-wcag/src/sia-r9/rule.ts

```diff
@@ -30,7 +30,12 @@ export const SIA_R9: Atomic.Rule<Document, Element> = {
     });
 
     expectations((target, expectation) => {
-      expectation(1, getRefreshTime(getAttribute(target, "content")!) === 0);
+      expectation(
+        1,
+        getRefreshTime(getAttribute(target, "content")!) === 0 ||
+          (getRefreshTime(getAttribute(target, "content")!) !== null &&
+            getRefreshTime(getAttribute(target, "content")!)! > 72000)
+      );
```

Comment:
Assign `getRefreshTime(getAttribute(target, "content")!)` to a variable in order to avoid re-parsing `content` thrice. In this case, `getRefreshTime()` also cannot return `null` as that has already been checked in the applicability of the rule.